pkg/controller/pkg/revision/reconciler.go

```diff
@@ -62,9 +62,9 @@ const (
 
 // Event reasons.
 const (
-	reasonParse   event.Reason = "ParsePackage"
-	reasonLint    event.Reason = "LintPackage"
-	reasonInstall event.Reason = "InstallPackage"
+	reasonParse  event.Reason = "ParsePackage"
+	reasonLint   event.Reason = "LintPackage"
+	reasonConfig event.Reason = "ConfigurePackage"
```

Comment:
nitpick: I don't love `ConfigurePackage` as a condition here, especially given our overloading of the term `Configuration`. Would certainly be open to alternatives to `InstallPackage` though

Reply:
I'm open to alternatives too - `InstallPackage` seemed inappropriate because it's sometimes used to effectively uninstall a revision - i.e. by removing a provider revision's deployment and configuration. I think elsewhere we've just used something like `ApplyPackage` if that seems any better.

Reply:
@negz we could say something like `SyncPackage`?